site/ja/tfx/guide/examplegen.md

```diff
@@ -0,0 +1,138 @@
+# ExampleGen TFX パイプラインコンポーネント
+
+Note: これらのドキュメントは私たちTensorFlowコミュニティが翻訳したものです。コミュニティによる
+翻訳は**ベストエフォート**であるため、この翻訳が正確であることや[英語の公式ドキュメント](https://www.tensorflow.org/?hl=en)の
+最新の状態を反映したものであることを保証することはできません。
+この翻訳の品質を向上させるためのご意見をお持ちの方は、GitHubリポジトリ[tensorflow/docs](https://github.com/tensorflow/docs)にプルリクエストをお送りください。
+コミュニティによる翻訳やレビューに参加していただける方は、
+[docs-ja@tensorflow.org メーリングリスト](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ja)にご連絡ください。
+
+ExampleGen はデータを TFX のパイプラインに投入するコンポーネントです。
+これは外部のファイルやサービスからデータを読み取り、ほかの TFX のコンポーネントが読み取る Examples を生成します。
+また、機械学習のベストプラクティスに従い、データセットの分割や並び替えについて、一貫性のある設定可能な方法で提供します。
+
+- 入力: CSV や `TFRecord`, BigQuery といった外部のデータソースからのデータ
+- 出力: `tf.Example` レコード
+
+## ExampleGen とその他のコンポーネント
+
+ExampleGen は [TensorFlow Data Validation](tfdv.md) を利用するコンポーネントにデータを提供します。
+これには [SchemaGen](schemagen.md), [StatisticsGen](statsgen.md), [Example Validator](exampleval.md) が含まれます。
+また、[TensorFlow Transform](tft.md) を利用する [Transform](transform.md) にもデータを提供し、最終的には推論時にデプロイメントターゲットへとデータを供給します。
+
+## ExampleGen コンポーネントの使い方
+
+サポートされるデータソース (現在、CSV ファイル、TF Example フォーマットの TFRecord ファイル、BigQuery にクエリした結果、の3つがサポートされています) を用いる場合、ExampleGen コンポーネントは典型的には非常に簡単にデプロイ可能で、ほとんど改修を必要とせずに利用できます。
+典型的なコードは次のようになります:
+
+```python
+from tfx.utils.dsl_utils import csv_input
+from tfx.components.example_gen.csv_example_gen.component import CsvExampleGen
+
+examples = csv_input(os.path.join(base_dir, 'data/simple'))
+example_gen = CsvExampleGen(input_base=examples)
+```
+
+次のように、外部の TF Example 形式のファイルを直接読み込むこともできます:
```

Comment:
[suggestion] `TF` -> `tf`

Reply:
This upper case is the same as original document. I guess it should be fixed at the original document. I'd like to left it.

Reply:
I see. In the page, three of four cases use `TF Example`.

Reply:
[question] If I take a look at the original markdown, it seems to be `or like below for importing external tf Examples directly:` .

Am I seeing the old one??

see: https://github.com/tensorflow/tfx/blob/master/docs/guide/examplegen.md